vhdbuilder/scripts/linux/mariner/tool_installs_mariner.sh

```diff
@@ -115,6 +115,98 @@ disableTimesyncd() {
     systemctl stop systemd-timesyncd || exit 1
     systemctl disable systemd-timesyncd || exit 1
     systemctl mask systemd-timesyncd || exit 1
+
+    # Update chrony configuration
+    cat > /etc/chrony/chrony.conf <<EOF
```

Comment:
is this a big change for AzLinux ? we were relying on the default config from the OS before I guess.

Reply:
It's a minor change, the config is 90% what was there before. The old ubuntu config file was copied from the Azure Linux default.